src/Controls/src/SourceGen/TypeConverters/ColumnDefinitionCollectionConverter.cs

```diff
@@ -30,7 +30,7 @@ public string Convert(string value, BaseNode node, ITypeSymbol toType, IndentedT
 			return $"new {columnDefinitionCollectionType.ToFQDisplayString()}([{string.Join(", ", columnDefinitions)}])";
 		}
 
-		context.ReportConversionFailed( xmlLineInfo, value, Descriptors.ColumnDefinitionCollectionConversionFailed);
+		context.ReportConversionFailed(xmlLineInfo, value, Descriptors.ColumnDefinitionCollectionConversionFailed);
```

Comment:
Revert this and other code formatting changes in this PR. Let's focus just on the task at hand.